util/env/env.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021-2024, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package env
+
+import (
+	"os"
+
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+// There are two CI steps, one to run tests using the path state scheme, and one to run tests using the hash state scheme.
+// An environment variable controls that behavior.
+func GetTestStateScheme() string {
```

Comment:
this function is only used for testing, so fits better in a "testhelpers" package (could be testhelpers/env.. but probably not worth it)

Reply:
I moved to "[github.com/offchainlabs/nitro/util/testhelpers/env](http://github.com/offchainlabs/nitro/util/testhelpers/env)" package.

It will be not straightforward to move this file to be part of the "[github.com/offchainlabs/nitro/util/testhelpers](http://github.com/offchainlabs/nitro/util/testhelpers)" package.
replay depends indirectly on testhelpers, and GetTestStateScheme indirectly depends on "[github.com/gofrs/flock](http://github.com/gofrs/flock)".
wasm doens't support some syscalls required by flock: 
```
~/Documents/offchain/git/nitro pathdb_fullnode⇣⇡* ❯ make test-go                                                                                           52s
mkdir -p `dirname [target/machines/latest/replay.wasm](http://target/machines/latest/replay.wasm)`
GOOS=wasip1 GOARCH=wasm go build -o [target/machines/latest/replay.wasm](http://target/machines/latest/replay.wasm) ./cmd/replay/...
# [github.com/gofrs/flock](http://github.com/gofrs/flock)
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:28:30: undefined: syscall.LOCK_EX
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:39:30: undefined: syscall.LOCK_SH
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:57:20: undefined: syscall.Flock
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:67:20: undefined: syscall.Flock
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:97:20: undefined: syscall.Flock
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:97:50: undefined: syscall.LOCK_UN
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:119:29: undefined: syscall.LOCK_EX
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:131:29: undefined: syscall.LOCK_SH
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:151:17: undefined: syscall.Flock
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:151:52: undefined: syscall.LOCK_NB
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/gofrs/flock@v0.8.1/flock_unix.go:151:52: too many errors
# [github.com/syndtr/goleveldb/leveldb/storage](http://github.com/syndtr/goleveldb/leveldb/storage)
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/syndtr/goleveldb@v1.0.1-0.20210819022825-2ae1ddf74ef7/leveldb/storage/file_storage.go:107:16: undefined: newFileLock
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/syndtr/goleveldb@v1.0.1-0.20210819022825-2ae1ddf74ef7/leveldb/storage/file_storage.go:192:3: undefined: rename
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/syndtr/goleveldb@v1.0.1-0.20210819022825-2ae1ddf74ef7/leveldb/storage/file_storage.go:267:12: undefined: rename
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/syndtr/goleveldb@v1.0.1-0.20210819022825-2ae1ddf74ef7/leveldb/storage/file_storage.go:272:12: undefined: syncDir
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/syndtr/goleveldb@v1.0.1-0.20210819022825-2ae1ddf74ef7/leveldb/storage/file_storage.go:555:9: undefined: rename
/home/diego/.gvm/pkgsets/go1.21/global/pkg/mod/github.com/syndtr/goleveldb@v1.0.1-0.20210819022825-2ae1ddf74ef7/leveldb/storage/file_storage.go:591:13: undefined: syncDir
make: *** [Makefile:273: [target/machines/latest/replay.wasm](http://target/machines/latest/replay.wasm)] Error 1
```